spring-cloud-generator/library_list.txt

```diff
@@ -1,90 +1,88 @@
-privateca, google/cloud/security/privateca/v1, com.google.cloud:google-cloud-security-private-ca, 758f0d1217d9c7fe398aa5efb1057ce4b6409e55
-servicecontrol, google/api/servicecontrol/v2, com.google.cloud:google-cloud-service-control, 758f0d1217d9c7fe398aa5efb1057ce4b6409e55
-servicedirectory, google/cloud/servicedirectory/v1, com.google.cloud:google-cloud-servicedirectory, 758f0d1217d9c7fe398aa5efb1057ce4b6409e55
-servicemanagement, google/api/servicemanagement/v1, com.google.cloud:google-cloud-service-management, 758f0d1217d9c7fe398aa5efb1057ce4b6409e55
-serviceusage, google/api/serviceusage/v1, com.google.cloud:google-cloud-service-usage, 758f0d1217d9c7fe398aa5efb1057ce4b6409e55
-cloudshell, google/cloud/shell/v1, com.google.cloud:google-cloud-shell, 758f0d1217d9c7fe398aa5efb1057ce4b6409e55
-speech, google/cloud/speech/v2, com.google.cloud:google-cloud-speech, 4c253358b1d4add3bf74707d5f58d44e044c5da8
-storagetransfer, google/storagetransfer/v1, com.google.cloud:google-cloud-storage-transfer, 758f0d1217d9c7fe398aa5efb1057ce4b6409e55
-jobs, google/cloud/talent/v4, com.google.cloud:google-cloud-talent, 758f0d1217d9c7fe398aa5efb1057ce4b6409e55
-cloudtasks, google/cloud/tasks/v2, com.google.cloud:google-cloud-tasks, 758f0d1217d9c7fe398aa5efb1057ce4b6409e55
-texttospeech, google/cloud/texttospeech/v1, com.google.cloud:google-cloud-texttospeech, a1b5429eec257c4f3a8425b20d4689d1df5812bc
-tpu, google/cloud/tpu/v2, com.google.cloud:google-cloud-tpu, 93c3e0eb6b4a4063daa4c9f89c488058699efe2b
-cloudtrace, google/devtools/cloudtrace/v2, com.google.cloud:google-cloud-trace, 758f0d1217d9c7fe398aa5efb1057ce4b6409e55
-videointelligence, google/cloud/videointelligence/v1, com.google.cloud:google-cloud-video-intelligence, 758f0d1217d9c7fe398aa5efb1057ce4b6409e55
-transcoder, google/cloud/video/transcoder/v1, com.google.cloud:google-cloud-video-transcoder, 758f0d1217d9c7fe398aa5efb1057ce4b6409e55
-vmmigration, google/cloud/vmmigration/v1, com.google.cloud:google-cloud-vmmigration, 758f0d1217d9c7fe398aa5efb1057ce4b6409e55
-vpcaccess, google/cloud/vpcaccess/v1, com.google.cloud:google-cloud-vpcaccess, 758f0d1217d9c7fe398aa5efb1057ce4b6409e55
-webrisk, google/cloud/webrisk/v1, com.google.cloud:google-cloud-webrisk, 758f0d1217d9c7fe398aa5efb1057ce4b6409e55
-websecurityscanner, google/cloud/websecurityscanner/v1, com.google.cloud:google-cloud-websecurityscanner, 758f0d1217d9c7fe398aa5efb1057ce4b6409e55
-workflowexecutions, google/cloud/workflows/executions/v1, com.google.cloud:google-cloud-workflow-executions, 807125e7953e32535710924fe4bf0362ac93754d
-workflows, google/cloud/workflows/v1, com.google.cloud:google-cloud-workflows, 758f0d1217d9c7fe398aa5efb1057ce4b6409e55
+# api_shortname, googleapis-folder, distribution_name:version, googleapis_committish, monorepo_folder
+accessapproval, google/cloud/accessapproval/v1, com.google.cloud:google-cloud-accessapproval, 758f0d1217d9c7fe398aa5efb1057ce4b6409e55, java-accessapproval
```

Comment:
Did you re-run the generation script for this? Is that why it got updated?

Reply:
Unintended and removed :)